src/tokens/formats/json-formats.js

```diff
@@ -0,0 +1,70 @@
+const jsonVariables = {
+  name: 'rvt/json/variables',
+  formatter: function(dictionary) {
+    const allProperties = dictionary.allProperties;
+    const properties = dictionary.properties;
+    // eslint-disable-next-line prefer-const
+    let categories = Object.keys(properties);
```

Comment:
You should be able to tack this on the front of your big `forEach` loop below. Might be a little cleaner and less verbose.